### PR TITLE
CORE-33547 - removing authoring mode config

### PR DESCRIPTION
**Pull request by artur-ciocanu**
Removing authoring mode config

## Description

Removing authoring mode config

## Motivation and Context

Removing authoring mode config

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All tests pass and I've made any necessary test changes.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: src/components/Personalization/index.js [hunk 1]
```diff
@@ -13,11 +13,13 @@ governing permissions and limitations under the License.
 import { isNonEmptyArray } from "../../utils";
 import { initRuleComponentModules, executeRules } from "./turbine";
 import { hideContainers, showContainers } from "./flicker";
-import { string, boolean } from "../../utils/configValidators";
+import { string } from "../../utils/configValidators";
 
 const DECISIONS_HANDLE = "personalization:decisions";
 const EVENT_COMMAND = "event";
 
+const isAuthoringMode = () => document.location.href.indexOf("mboxEdit") !== -1;
```
Comment (jfkhoury) [inline src/components/Personalization/index.js]: Let's add a comment in here pls.

**Comment by Aaronius** [comment]
This stems from: https://github.com/AdobeDocs/alloy-docs/pull/31#discussion_r313952248 and other discussion @jfkhoury's been in on. Optimally, I would prefer we use a query string parameter that's Alloy-specific (e.g., `alloy_authoring_enabled=true`) but that requires changes to the visual editor.

**Comment by artur-ciocanu** [comment]
@Aaronius that's the plan once this is approved, I'll open a PR for docs.

File: src/components/Personalization/index.js (hunk 1 above)
Comment (artur-ciocanu) [reply src/components/Personalization/index.js]: Done

**Pull request merged**